include/net/fota_download.h

```diff
@@ -105,6 +107,14 @@ int fota_download_init(fota_download_callback_t client_callback);
 int fota_download_start(const char *host, const char *file, int sec_tag,
 			const char *apn, size_t fragment_size);
 
+/**@brief Cancel FOTA image downloading.
+ *
+ * @retval 0 If FOTA is cancelled successfully.
+ *         -EALREADY If download is not started, aborted or completed.
```

Comment:
I believe this needsa separate `@retval` entry (if we list the possible return values).

Reply:
Added